### PR TITLE
Add support for tabular aliases

### PR DESCRIPTION
**Pull request by inferrinizzard**
alias tabular mode aligns aliases and potential AS token to length of longest column / expression +1

TODO: add edge case logic for newline: never and keywordPosition: tenSpace + tests

### CODE REVIEW

File: src/core/Formatter.ts [hunk 1]
```diff
@@ -85,9 +85,59 @@ export default class Formatter {
...
+
+		return query;
+	}
+
+	formatAliasPositions(query: string) {
+		const lines = query.split('\n');
+
+		let newQuery: string[] = [];
+		for (let i = 0; i < lines.length; i++) {
+			// find SELECT rows with trailing comma, if no comma (only one row) - no-op
```
Comment (inferrinizzard) [inline src/core/Formatter.ts:107]: add edge case for NewlineMode.never and tenSpace keywordPosition modes

**Review by inferrinizzard — COMMENTED** [review]
Needs edges cases for future integrations,
ignoring table aliases
👍
